src/wdio/services/TerraCommands/visual-regression.js

```diff
@@ -74,13 +72,14 @@ const themeCombinationOfCustomProperties = (...args) => {
 
   const selector = args[0].selector ? args[0].selector : global.browser.options.terra.selector;
   const styleProperties = args[0].properties ? args[0].properties : [];
-
+  let testName;
```

Comment:
Lets follow the fallback syntax we've been doing throughout terra-toolkit:
```diff
+  const options = args[0];
+  const selector = options.selector || testName;
+  const selector =options.selector || global.browser.options.terra.selector;
-  const selector = args[0].selector ? args[0].selector : global.browser.options.terra.selector;
+  const styleProperties = options.properties || [];
-  const styleProperties = args[0].properties ? args[0].properties : [];
```

Reply:
I have updated the syntax. GitHub link: https://github.com/cerner/terra-toolkit/pull/276/commits/f2db2d82b31eca7f6d3e237e2836019874bdeaae